sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -1279,4 +1278,64 @@ class CSVSuite extends QueryTest with SharedSQLContext with SQLTestUtils {
       Row("0,2013-111-11 12:13:14") :: Row(null) :: Nil
     )
   }
+
+  val sampledTestData = (row: Row) => {
```

Comment:
I have to define it as `val` because `def` contains a reference to outer class `CSVSuite` and the lambda inside of `map(...)` becomes not serializable.